book/website/ethical-research/law-policy.md

```diff
@@ -0,0 +1,251 @@
+If you do not explicitly license your content differently, it generally falls under copyright and technically cannot be remixed without your permission. 
+This can present problems for working openly!
+
+You may use an open license to share code, or content, or both. 
+There are lots of licenses designed especially for sharing broadly, and organizations like Creative Commons and the Free Software Foundation have created tools and resources to facilitate sharing of content, data and code. 
+There is more information in the [Licensing](https://the-turing-way.netlify.app/reproducible-research/licensing.html) chapter.
+
+#### Tort law
+
+Tort law is the branch of law which allows people who have been harmed to sue for compensation or other remedy for that harm.
+Tort law varies by jurisdiction. 
+It can include both harm that arises from negligence, and intentional harm. 
+If you harm someone, either intentionally or negligently, you may be legally liable for the harm caused.
+
+Acting ethically, however, usually means going beyond thinking only about legal liability. 
+You should be aware of all the possible harm that your work could cause, not only what you might be legally liable for. 
+
+#### Contract law
+
+Contract law is the branch of law which handles how contracts - legal agreements between parties - are enforced.
+You should know the terms of contracts you have signed, and what those terms mean.
+You should also know which jurisdiction covers the contract you have signed (this should be in the contract). 
+
+Ethical work can also include thinking critically about contracts.
+You should think about what obligations you are signing up for when you sign a contract.
+You should also think about what contractual obligations you are imposing on other people when you form a contract with them.
+
+### How to change laws
+
+Law is not static and unchanging. 
+The law is made by humans and it can be changed by humans.
+If you feel that the law in your jurisdiction works in a way that is unhelpful or damaging, consider joining efforts to change it!
+Changing laws is a long-term project and can take decades. 
+This section is not intended to explain how to do this: a good place to start is to find a campaigning organisation who is working on the issue, and ask them how to get involved. 
+<!--- edit this to refer to the activism chapter, when we have written it --> 
+
+## Policy
+
+Policy is a general term for a set of ideas or principles that guide actions in particular situations.
+Policies are not legally binding, but they are designed to encourage certain kinds of behaviours and discourage others.
+
+Policies are agreed between groups of people: this can be at organisational, intra-organisational, or governmental level. 
+It is useful to be aware, for example, of policies on ethical research and innovation that are set by your government, or your funding organisation.
+
```

Comment:
I think that it would be helpful to give concrete examples of policies that relate to Ethics. I would also mention Codes of Conduct as policies that help in RRI and RI, as mentioned in the book's introduction. I would start by giving examples of The Turing Way policies. For example, the [ethical guidelines for statistical practice of the American Statistical Association](https://www.amstat.org/ASA/Your-Career/Ethical-Guidelines-for-Statistical-Practice.aspx) could be another example.

Reply:
```suggestion
For example, the American Statistical Association has created [ethical guidelines for statistical practice](https://www.amstat.org/ASA/Your-Career/Ethical-Guidelines-for-Statistical-Practice.aspx).
```

Reply:
great idea to add TTW code of conduct! I have also added the ASA policy.